firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -270,7 +270,6 @@ class SearchTests: BaseTestCase {
             // Reload icon is displayed.
             mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton])
             XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton].label, "New Tab")
-            navigator.performAction(Action.CloseURLBarOpen)
```

Comment:
@mdotb-moz We may need to put this back after we get an answer about the behaviour of the keyboard.
https://github.com/mozilla-mobile/firefox-ios/issues/23436